kotori/daq/decoder/tts_ttn.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# (c) 2022 Andreas Motl <andreas@getkotori.org>
+import json
+from collections import OrderedDict
+
+
+class TheThingsStackDecoder:
+    """
+    Decode JSON payloads in TTS-/TTN-webhook JSON format.
+    TTS/TTN means "The Things Stack" / "The Things Network".
+
+    Documentation
+    =============
+    - https://getkotori.org/docs/handbook/decoders/tts-ttn.html
+
+    Development
+    ===========
+    - https://community.hiveeyes.org/t/more-data-acquisition-payload-formats-for-kotori/1421
+    - https://community.hiveeyes.org/t/ttn-daten-an-kotori-weiterleiten/1422/34
+    """
+
+    @staticmethod
+    def decode(payload: str):
+
+        # Decode from JSON.
+        message = json.loads(payload)
+
+        # Use timestamp and decoded payload.
+        data = OrderedDict()
+        data["timestamp"] = message["received_at"]
+        data.update(message["uplink_message"]["decoded_payload"])
+
+        # TODO: Add more data / metadata.
+        #       This is an implementation from scratch. It can be improved by
+        #       cherry-picking more specific decoding routines from `ttnlogger`.
+        #       https://github.com/daq-tools/ttnlogger
```

Comment:
@thiasB: This is the gist of decoding TTN uplink messages which would then happen inside Kotori. I can remember that we crafted an advanced version of this within [ttnlogger](https://github.com/daq-tools/ttnlogger) the other day, right?

So, when we are aiming to get en par with the implementation from `ttnlogger`, we would need to fill some gaps here.